### DISCUSSION TITLE
Name connection between the service and rdb

### DISCUSSION THREAD

**Discussion by lzd-1230** [ceph/ceph-csi#4265] [Q&A]
Hi guys, How can I add pvc information to the rdb, so that I can handle rdb more clear. Not like the default setting when i using a PVC such as `ldap-data-0` but finally in rdb it create a image named `csi-vol-60eab81b-35f1-48cb-bee7-ee8d7b10ecdd` which make no sense in ceph management panel . I've seen the `volumeNamePrefix` option in storageclass, but this is static prefix, not a dynamic connection...
Is there a best way to link the name between the service and rdb? Thanks for tutoring!

**Comment by nixpanic** [comment]
If the Ceph-CSI executables have the `--setmetatada=true` option on the commandline, some Kubernetes identification labels are set in the metadata of the RBD image. You can check the metadata with the `rbd image-meta list pool/image` command.
